tests/api/test_subscriber.py

```diff
@@ -153,6 +154,81 @@ def test_get_subscriber(self, mock_request: mock.MagicMock) -> None:
             timeout=5,
         )
 
+    @mock.patch("requests.request")
+    def test_get_subscriber_with_credentials_info(self, mock_request: mock.MagicMock) -> None:
+        mock_request.return_value = MockResponse(
+            200,
+            {
+                "data": {
+                    "_id": "63dafedbc037e013fd82d37a",
+                    "_organizationId": "63dafed97779f59258e3843f",
+                    "_environmentId": "63dafed97779f59258e38445",
+                    "subscriberId": "63dafed4117f8c850991ec4a",
+                    "channels": [
+                        {
+                            'provider_id': 'slack',
+                            '_integration_id': '64f6d74be166fcd7f2751111',
+                            'credentials': {
+                                'webhook_url': 'TEST',
+                                'channel': 'slack',
+                                'device_tokens': ["TEST"]
+                            },
+                            'integration_identifier': None
+                        }
```

Comment:
For me, Novu use lowerCamelCase in his response.

An utile in the lib is responsible for the conversion from lowerCamelCase to snake_case

```suggestion
                            'providerId': 'slack',
                            '_integrationId': '64f6d74be166fcd7f2751111',
                            'credentials': {
                                'webhookUrl': 'TEST',
                                'channel': 'slack',
                                'deviceTokens': ["TEST"]
                            },
                            'integrationIdentifier': None
                        }
```